src/Tribe/RSVP.php

```diff
@@ -314,6 +315,12 @@ public function render_rsvp_step( $ticket_id, $step = null ) {
 			return '';
 		}
 
+		// Check to make sure post is not deleted, trashed or draft.
+		$post_status = get_post_status( $post_id );
+		if ( 'publish' !== $post_status ) {
```

Comment:
Shouldn't they be able to RSVP for private events for which they have the right to see ? 

Maybe we should check if `current_user_can` instead.